### PR TITLE
For color, use "sRGB color hex triplet" (P465) in priority and "color…

### LINKED ISSUE #5: For color, use "sRGB color hex triplet" (P465) in priority and "color" (P462) as a fallback
Some parties have a "sRGB color hex triplet" (P465) already defined in Wikidata.
This should be used if defined. If not, use the "color" (P462).

### PR DESCRIPTION
**Pull request by jonadem**
…" (P462) as a fallback

Fix #5

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -75,15 +89,23 @@ const init = () => {
 
         return;
     }
-    getData(values.parliament, values.term).then(data => {
-        const parties = data.reduce((acc, party) => {
+    getData(values.parliament, values.term, values.lang).then(data => {
```
Comment (k-nut) [inline app/app.js]: I suppose you just forgot to remove this `values.lang` here? It seems like we don't actually do anything with it.

**Comment by k-nut** [comment]
Thanks, this seems like a good idea. I slightly refactored your code, let me know if you find any issues with that.

BTW: I've seen for the members in Germany that sometimes the fraction-qualifiers on their membership properties point to items that describe a political party (such as `SPD`), sometimes to items that describe political fractions (such as `Fraktion CDU/CSU im Bundestag`) and sometimes to items that describe political fractions for certain time periods (such as `Fraktion die Linke im 15. Landtag von Hessen`). Is there any consensus in Belgium already which style would be preferred?

**Comment by jonadem** [comment]
It's better with your refactoring (Indeed, I forgot to clean the "lang" arg).

For the properties, my preference goes to the political fraction (such as `Fraktion CDU/CSU im Bundestag`) but sometimes I'm lazy and I add the political party (such as `SPD`). I find the political fractions for certain time periods (such as `Fraktion die Linke im 15. Landtag von Hessen`) very limitating. This is not a consensus, it's just my personal feeling. ;-)

**Comment by k-nut** [comment]
Ok, cool. I actually share your feeling regarding the properties and will try to follow this pattern form now on :)

**Pull request merged**